testing/trino-benchto-benchmarks/src/test/resources/sql/presto/tpch/q09.plan.txt

```diff
@@ -12,11 +12,11 @@ remote exchange (GATHER, SINGLE, [])
                                             join (INNER, PARTITIONED):
                                                 remote exchange (REPARTITION, HASH, ["suppkey_5"])
                                                     join (INNER, PARTITIONED):
-                                                        remote exchange (REPARTITION, HASH, ["partkey"])
```

Comment:
why these flipped? They were unestimated?

Reply:
The `p.name LIKE '%green%'` filter on `part` is unestimated. `part` size is `3.35GB` and `lineitem` size is `340.72GB`. So the existing logic left this alone, now we take the decision of flipping sides, without changing join distribution type to REPLICATED, based on the large difference in sizes.